keras_nlp/src/tokenizers/sentence_piece_tokenizer.py

```diff
@@ -172,6 +176,8 @@ def set_proto(self, proto):
         self._sentence_piece = tf_text.SentencepieceTokenizer(
             model=proto_bytes,
             out_type=self.compute_dtype,
+            add_bos=self.add_bos,
```

Comment:
Looks good! We will need to propagate this in `get_config()` below, so this will persist with Keras' serialization and saving routines.